pyproject.toml

```diff
@@ -11,6 +11,9 @@ click = "^8.1.7"
 gcn-kafka = "^0.3.3"
 python = "^3.9"
 prometheus-client = "^0.21.0"
+boto3 = "^1.35.7"
```

Comment:
just out of curiosity, do we need to use 1.35.7 for a specific reason? current version is 1.35.53 so I was just wondering.

Reply:
Nope, I think it was just the version that I had at the time